content/pillow/concepts/image/terms/merge/merge.md

```diff
@@ -0,0 +1,49 @@
+---
+Title: '.merge()'
+Description: 'Set of single band images into a new multiband image'
```

Comment:
As per the [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md), `Description`: for a coding term should begin with a present-tense verb, i.e., "Returns", "Calculates", "Outputs", etc.

```suggestion
Description: 'Merges set of single-band images into a new multi-band image.'
```